src/Sylius/Component/Core/Resolver/DefaultShippingMethodResolver.php

```diff
@@ -25,6 +25,8 @@
 use Sylius\Component\Shipping\Resolver\DefaultShippingMethodResolverInterface;
 use Webmozart\Assert\Assert;
 
+@trigger_error('This class is deprecated since Sylius 1.2 and will be removed in 2.0. "Sylius\Component\Core\Resolver\CategoryBasedDefaultShippingMethodResolver" should be used instead.', E_USER_DEPRECATED);
```

Comment:
`"Sylius\Component\Core\Resolver\CategoryBasedDefaultShippingMethodResolver"` screams for `sprintf("%s", CategoryBasedDefaultShippingMethodResolver::class)`